notes/v1.0.0-M11.md

```diff
@@ -0,0 +1,229 @@
+# Install the latest release :candy:
+
+If you're on Mac OS X, upgrade to the latest version with:
+
+```sh
+$ brew install scalacenter/bloop/bloop
+```
+
+Otherwise, run:
+
+```
+$ curl -L https://github.com/scalacenter/bloop/releases/download/v1.0.0-M11/install.py | python
+```
+
+Read the complete instructions in our [Installation page][installation instructions].
+
+# Highlights :books:
+
+## [Redesign the configuration format][#529] (by [Jorge Vicente Cantero][@jvican])
+
+With Bloop 1.0 being around the corner, we need to fix all the issues with the current configuration
+file format, and come up with a solution that will be stable for a long time.
+
+Jorge has been working towards a definitive design for what the configuration file should look like,
+and his intensive head scratching resulted in this pull request.
+
+## Mill integration (by [Olivier Mélois][@Baccata]) :factory:
+
+Bloop now includes a [Mill integration][#532], which means that your build definition can be exported from
+Mill's format to Bloop's.
+
+To try the Mill integration, add the following to the `build.sc` of a project:
+
+```scala
+import $ivy.`ch.epfl.scala::mill-bloop:1.0.0-M11`
+```
+
+And then run `mill bloop.integrations.mill.Bloop/install` from the directory of the mill project.
```

Comment:
Could be `mill.Bloop/install` instead if you move the class to another package?

Reply:
I imagine so, I admit I have no experience with mill... These instructions come from #532 

PR welcome :)

Reply:
I would consider it if I used mill. Cc/ @baccata

Reply:
I wanted to be consistent with how the sbt integration is packaged. Personally I just have a top level task that delegate to that module so I just have to call `mill bloop`, so I'm not really annoyed by that.